sys/benchmark/benchmark.c

```diff
@@ -27,6 +27,10 @@ void benchmark_print_time(uint32_t time, unsigned long runs, const char *name)
     uint32_t full = (time / runs);
     uint32_t div  = (time - (full * runs)) / (runs / 1000);
 
-    printf("%11s: %9" PRIu32 "us  ---  %2" PRIu32 ".%03" PRIu32 "us per call\n",
-           name, time, full, div);
+    uint32_t per_sec = (uint32_t)(((uint64_t)1000000UL * runs) / time);
+
+    printf("%25s: %9" PRIu32 "us"
+           "  ---  %2" PRIu32 ".%03" PRIu32 "us per call"
+           "  ---  %9" PRIu32 " calls per sec\n",
+           name, time, full, div, per_sec);
```

Comment:
Does this change require updates to any pexpect scripts?

Reply:
yes, I've pushed a fix to Hauke's branch